services/response_code.proto

```diff
@@ -1373,4 +1373,19 @@ enum ResponseCodeEnum {
    * An alias that is assigned to an account or contract cannot be assigned to another account or contract.
    */
   ALIAS_ALREADY_ASSIGNED = 332;
+
+  /**
+   * A provided metadata key was invalid. Verify the bytes for an Ed25519 public key are exactly 32 bytes; and the bytes for a compressed ECDSA(secp256k1) key are exactly 33 bytes, with the first byte either 0x02 or 0x03..
```

Comment:
Keys can have complex structures; so the verification listed here might be inapplicable or at least incomplete. 

Maybe just:
```
A provided metadata key was invalid. Verification includes, for example, checking the size of Ed25519 and ECDSA(secp256k1) public keys. 
```

Reply:
Done. Thanks @tinker-michaelj